packages/adminrouter/extra/src/test-harness/modules/generic_test_code/common.py

```diff
@@ -71,6 +74,8 @@ def generic_upstream_headers_verify_test(ar, auth_header, path, assert_headers=N
         path (str): path for which request should be made
         assert_headers (dict): additional headers to test where key is the
             asserted header name and value is expected value
+        assert_header_absent (dict): headers that *MUST NOT* be present in the
```

Comment:
❓ The type isn't `dict` it is `dict or NoneType`. In type hints this might be:

```python
from typing import Dict, Union

assert_headers_absent: Union[Dict, None]
```